src/pages/memberhsip/components/youthProfileForm/YouthProfileForm.module.css

```diff
@@ -44,10 +48,34 @@
     margin-bottom: 10px;
 }
 
-.checkBoxLabel {
-    margin-left: 5px;
+.radioButtonRow {
+    float: left;
+    margin-right: 50px;
+}
+
+.listLabel {
+    margin-left: 10px;
+}
+
+.hidePhotoPermit {
+    display:none;
 }
 
 .acceptTermsCheckBox {
     padding-left: 0;
+}
+
+a {
+    text-decoration: none;
+}
+
+button {
+    margin-top: 30px;
+    height: 50px;
+    width: 400px;
+}
+
+.buttonEnabled {
```

Comment:
It would be better to incorporate these enabled styles to button, and make disabled styles with `button:disabled { ... }`